pkg/dao/rpms.go

```diff
@@ -529,8 +530,8 @@ func (r *rpmDaoImpl) DetectRpms(orgID string, request api.DetectRpmsRequest) (*a
 	}
 
 	// find rpms associated with the repositories that match given rpm names
-	orGroupPublicOrPrivate := r.db.Where("repository_configurations.org_id = ?", orgID).Or("repositories.public")
-	db := r.db.
+	orGroupPublicOrPrivate := r.db.WithContext(ctx).Where("repository_configurations.org_id = ?", orgID).Or("repositories.public")
```

Comment:
is `WithContext` needed in this where clause since you're calling it again below? i'm not sure if it hurts to add it multiple times or not

Reply:
It probably isn't needed here only the outer most r.db, but wouldn't hurt anything.  However i added it by mistake here so i'll remove it.